components/Launchpad/lauchpad.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useEffect, useRef, useState} from 'react';
+import 'react-native-gesture-handler';
+import {
+  Button,
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  TextInput,
+  useColorScheme,
+  View,
+} from 'react-native';
+
+import {useAsyncStorage} from '@react-native-async-storage/async-storage';
+
+import {Ang} from '../../components';
+import {layoutStyles, elementStyles} from '../../styles';
+
+const Launchpad = (): JSX.Element => {
+  const isDarkMode = useColorScheme() === 'dark';
+  const textInputRef = useRef<TextInput>(null);
+
+  const [inputAng, setInputAng] = useState(1);
+  const {getItem, setItem} = useAsyncStorage('@currentAng');
+
+  const readItemFromStorage = async () => {
+    const item = await getItem();
+    if (item) {
+      setInputAng(parseInt(item, 10));
+    }
+  };
+
+  const saveCurrentAng = async (newValue: number) => {
+    await setItem(newValue.toString());
+    setInputAng(newValue);
+  };
+
+  useEffect(() => {
+    readItemFromStorage();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
this can be fixed. Similar issue mentioned above

Reply:
I tried this, and we even had a call about this, fixing the linting issue, makes this useEffect run on every change, we only want it to run once.